test/SILOptimizer/Inputs/SpanExtras.swift

```diff
@@ -0,0 +1,1140 @@
+import Builtin
+
+@_unsafeNonescapableResult
+@inlinable @inline(__always)
+@lifetime(borrow source)
+public func _overrideLifetime<
```

Comment:
I would be helpful to use the same syntax for _overrideLifetime everywhere. We should copy the standard library's implementation. I always spend a lot of time adapting code from @glessard because he hasn't updated his repos yet.

`_overrideLifetime(_:borrowing)`
`_overrideLifetime(_:copying)`
`_overrideLifetime(_:mutating)`